scalanet/src/io/iohk/scalanet/peergroup/kademlia/KRouter.scala

```diff
@@ -500,9 +519,25 @@ object KRouter {
   // sequence number (why)
   // compressed public key (why)
   // TODO understand what these things do, which we need an implement.
-  case class NodeRecord[A](id: BitVector, routingAddress: A, messagingAddress: A) {
+  case class NodeRecord[A](id: BitVector, routingAddress: A, messagingAddress: A,sec_number:Long,sign:(BigInteger,BigInteger)) {
```

Comment:
why here? I mean, maybe now we can move this case class, with the companion object, to a different file.